### ISSUE TITLE
sql: textual EXPLAIN(distsql) output

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/cockroach#50122]
EXPLAIN(vec) currently produces a textual display of the vectorized operators in a distributed plan. 

This issue tracks producing a textual display of the DistSQL specs in a distributed plan, so that it's possible to understand a plan without having to read the graphical output, which can be hard to parse visually the bigger the plan gets.

As a first cut, we should aim to use the same tree-creation method as EXPLAIN(vec).

The URL output can be displayed as the first row in the output, or moved to a different command altogether, TBD.

**Comment by awoods187** [comment]
This work is closely related to https://github.com/cockroachdb/cockroach/issues/41459

**Comment by yuzefovich** [comment]
@RaduBerinde it looks like you'll be addressing this issue soon?

**Comment by RaduBerinde** [comment]
For now, we are going in a slightly different direction - we will annotate the EXPLAIN logical plan tree with information related to distribution, so that the diagram doesn't need to be consulted in most cases.
